app/com/arpnetworking/metrics/portal/alerts/impl/AlertExecutionCacheActor.java

```diff
@@ -366,7 +385,7 @@ public Builder setEntries(final List<SnapshotEntry> entries) {
 
     private static final class SnapshotEntry implements AkkaJsonSerializable {
         private final CacheKey _key;
-        private final JobExecution.Success<AlertEvaluationResult> _value;
```

Comment:
In general ObjectMapper should be able to get the type parameter from the type definition since it's static. Maybe this has to do with how we set up our mapper